### PR TITLE
Extracted MQTT and ping intervals to constants

### PR DESCRIPTION
**Pull request by djesic**

### CODE REVIEW

File: wolk/OSMQTTConnectivityService.py [hunk 1]
```diff
@@ -223,7 +236,9 @@ def connect(self):
             self.device.key,
             self.device.password,
         )
-        self.client.connect(self.host, self.port)
+        self.client.connect(
```
Comment (srdjastankovic) [inline wolk/OSMQTTConnectivityService.py:239]: It can be overwrite with call?

File: wolk/WolkConnect.py [hunk 2]
```diff
@@ -276,17 +303,19 @@ def __init__(
                     ca_cert=wolk_ca_cert,
                 )
 
-        self.connectivity_service.set_inbound_message_listener(self._on_inbound_message)
+        self.connectivity_service.set_inbound_message_listener(
```
Comment (srdjastankovic) [inline wolk/WolkConnect.py:306]: You applied new formating?

Comment (djesic) [reply wolk/WolkConnect.py:306]: new formatting, yes

File: wolk/OSMQTTConnectivityService.py (hunk 1 above)
Comment (djesic) [reply wolk/OSMQTTConnectivityService.py:239]: oops

Comment (djesic) [reply wolk/OSMQTTConnectivityService.py:239]: it is overwritten, check next line!

**Pull request merged**